example/requirements.txt

```diff
@@ -1,3 +1,4 @@
+botocore==1.29.165
```

Comment:
botocore dependency in pynamodb is not sufficiently strict to work with pydjamodb